api/feedbooks.py

```diff
@@ -89,7 +89,7 @@ def __init__(self, _db, collection, *args, **kwargs):
 
         self.new_css = None
         if new_css_url and self.http_get:
-            status_code, headers, content = self.http_get(new_css_url)
+            status_code, headers, content = self.http_get(new_css_url, {})
```

Comment:
An unrelated change that caused a real-world Feedbooks import to fail even though it worked when `http_get` was mocked.